router.go

```diff
@@ -50,7 +50,11 @@ func main() {
 			log.Printf("Failed to write new nginx configuration; continuing with existing configuration: %v", err)
 			continue
 		}
-		nginx.Reload()
+		err = nginx.Reload()
```

Comment:
Fair enough. I just followed what the rest of the file was doing. We'll switch over to the one-liner in another PR.